contracts/interchain-token-service/src/testutils.rs

```diff
@@ -0,0 +1,3 @@
+#![cfg(any(test, feature = "testutils"))]
+pub const INTERCHAIN_TOKEN_WASM_HASH: &[u8] =
+    include_bytes!("../tests/testdata/interchain_token.wasm");
```

Comment:
```suggestion
    include_bytes!("./testdata/interchain_token.wasm");
```
move the file to `src/testdata` and update ITS tests to use this as well